lua/nvim-highlight-colors/color/utils_spec.lua

```diff
@@ -182,12 +182,20 @@ describe('Color Utils', function()
 		assert.are.equal(hsl_table[1], '240')
 		assert.are.equal(hsl_table[2], '100')
 		assert.are.equal(hsl_table[3], '68')
+		local hsl_table_decimal = utils.get_hsl_values("hsl(240.5, 100.5%, 68.5%)")
+		assert.are.equal(hsl_table[1], '240.5')
+		assert.are.equal(hsl_table[2], '100.5')
+		assert.are.equal(hsl_table[3], '68.5')
 	end)
 
 	it('should return hsl color properties when receiving css variable name', function()
 		local hsl_table = utils.get_hsl_without_func_values("--name: 0 0% 100%;")
 		assert.are.equal(hsl_table[1], '0')
 		assert.are.equal(hsl_table[2], '0')
 		assert.are.equal(hsl_table[3], '100')
+		local hsl_table = utils.get_hsl_without_func_values("--name: 10.5 20.5% 80.5%;")
+		assert.are.equal(hsl_table[1], '10.5')
+		assert.are.equal(hsl_table[2], '20.5')
+		assert.are.equal(hsl_table[3], '100.5')
```

Comment:
```suggestion
		assert.are.equal(hsl_table[3], '80.5')
```

Reply:
yeah, the lightness value cannot exceed 100